ee_tests/local_cico_run_EE_tests_ts.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Show command before executing
+set -x
```

Comment:
This exposes passwords and tokens - even though this is intended to be used locally in safe environment, I suggest not to expose passwods and tokens, just need to move this command after sourcing `local_osio.conf.sh`